pkgs/applications/window-managers/hyprwm/hyprland/wlroots.nix

```diff
@@ -51,20 +22,11 @@ wlroots.overrideAttrs
       + "-hyprland"
       + lib.optionalString enableNvidiaPatches "-nvidia";
 
-    patches =
-      (old.patches or [ ])
-      ++ (lib.optionals enableNvidiaPatches [
-        "${hyprland.src}/nix/patches/wlroots-nvidia.patch"
-      ]);
+    patches = lib.optionals enableNvidiaPatches [
+      "${hyprland.src}/nix/patches/wlroots-nvidia.patch"
+    ];
 
-    # don't need old.postPatch for hwdata's path in wlroots 0.16
     postPatch = lib.optionalString enableNvidiaPatches ''
       substituteInPlace render/gles2/renderer.c --replace "glFlush();" "glFinish();"
```

Comment:
I am not a fan of editing C code. It is better to use a patch file.

Reply:
That's no longer relevant. It's being removed as of https://github.com/NixOS/nixpkgs/pull/272333.